.github/pull_request_template.md

```diff
@@ -1,4 +1,17 @@
-### Please remember to
+### What does this PR do?
 
-- [ ] Give the PR a meaningful title -- it will be shown in the release notes
-- [ ] Add (or remove) [user questions](https://github.com/giantswarm/docs/blob/main/CONTRIBUTING.md#front-matter) associated with any updated docs
+(Please set a descriptive PR title. Use this space for additional explanations.)
+
+### How does it look like?
```

Comment:
```suggestion
### How does it look?
```
or
```
### What does it look like?
```

Reply:
Going for "What does it look like?". Thanks!